src/apps/btc/btc.c

```diff
@@ -26,6 +26,9 @@
 #include <workflow/status.h>
 #include <workflow/verify_pub.h>
 
+#define BTC_TOP_IDENTIFIER (451509 + BIP32_INITIAL_HARDENED_CHILD)
```

Comment:
maybe it's just me but i find the naming very non-descriptive. please add comments.